README.md

```diff
@@ -62,5 +62,8 @@ $ make
 $ sudo make install
 ```
 
+# Distribution Packages
+[Arch Linux](https://aur.archlinux.org/packages/woeusb-git/)
+
 ## License
-WoeUSB is sold under the [GPL license](https://github.com/slacka/WoeUSB/blob/master/COPYING).
+WoeUSB is sold under the [GPL license](https://github.com/slacka/WoeUSB/blob/master/COPYING).
```

Comment:
Thanks for your contribution, however it seems that the last line is modified in this commit, please exclude it if possible, thanks!